hasher-matcher-actioner/webapp/src/components/ContentMatchPane.jsx

```diff
@@ -36,43 +38,61 @@ export default function ContentMatchPane({contentId, signalId, signalSource}) {
       });
   }, [contentId]);
 
+  useEffect(() => {
+    fetchContentDetails(contentId).then(result => {
+      // Ensure placeholders don't get displayed. TODO: Move this to the API.
+      const additionalFields = result.additional_fields.filter(
+        x => x !== 'Placeholder',
+      );
+
+      setContentDetails({...result, additional_fields: additionalFields});
+    });
+  }, [contentId]);
+
   return (
     (hashDetails === null && <p>Loading...</p>) || (
       <Container>
         <Row>
           {/* Avoid explicit padding if possible when re-doing this page. */}
-          <Col className="p-4">
+          <Col className="p-4 text-center" style={{minHeight: '200px'}}>
             <BlurUntilHoverImage src={img} />
           </Col>
         </Row>
         <Row>
-          <Col md={3} className="p-4">
-            Details
-          </Col>
           <Col className="p-4">
-            <Link to={`/matches/${contentId}`}>
-              View Details for this matched content.
-            </Link>
-          </Col>
-        </Row>
-        <Row>
-          <Col md={3} className="p-4">
-            Signal
-          </Col>
-          <Col className="p-4">
-            <Link to={`?signalId=${signalSource}|${signalId}`}>
-              View Matches for Signal
-            </Link>
-          </Col>
-        </Row>
-        <Row>
-          <Col md={3} className="p-4">
-            Content
-          </Col>
-          <Col className="p-4">
-            <Link to={`?contentId=images/${contentId}`}>
-              View Matches of this Content
-            </Link>
+            <h4 className="mt-4">Content Details</h4>
+            <p>
+              Last Submitted to HMA:{' '}
+              <b>
+                {contentDetails && contentDetails.updated_at
+                  ? formatTimestamp(contentDetails.updated_at)
+                  : 'Unknown'}
+              </b>
+            </p>
+
+            <h4 className="mt-4">Additional Fields</h4>
+            {contentDetails && contentDetails.additional_fields.length !== 0 ? (
+              contentDetails.additional_fields.map(field => <li>{field}</li>)
+            ) : (
+              <p>No additional fields provided</p>
+            )}
+
+            <p className="mt-4">
+              <Link to={`/matches/${contentId}`}>See all details</Link>
+            </p>
+
+            <h4 className="mt-4">Filter search results</h4>
+            <li>
+              <Link to={`?signalId=${signalSource}|${signalId}`}>
+                Show matches for this signal
```

Comment:
What about something like making this or a part of it a button? 
e.g.
"Show only the matches for this signal: <Button>Apply Filter</Button>"